src/content/general/organizations/index.md

```diff
@@ -0,0 +1,141 @@
+
+## Organization CRD and CRs {#organization-crd-cr}
+
+If the concept of custom resources (CR) and custom resource definitions (CRD) is new to you: Kubernetes allows to define [arbitrary objects](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to be handled via the Kubernetes API. The schema of such an object is specified by a custom resource definition. The actual objects are called the custom resources.
+
+Giant Swarm management clusters provide a CRD named `Organization` (long form: `organizations.security.giantswarm.io`). An organization is defined simply by a custom resource using that CRD, which we'll call an "organization CR" here for brevity.
+
+> An organization is defined by an organization CR.
+
+Our CRD schema documentation provides details about the [Organization CRD]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}). But before you raise your eyebrows in disappointment, be warned: there isn't much to document. The single most important aspect of an organization CR is it's name. But there is more to it, of course.
+
+Once an organization CR is created, our automation ([organization-operator](https://github.com/giantswarm/organization-operator), to be precise) ensures that a namespace exists for the organization. More about that [in a minute](#namespace).
+
+## Naming conventions {#naming-conventions}
+
+Organization names (technically: organization CR names) must follow these rules:
+
+- Must be unique within the management cluster
+- Must contain at most 59 characters.
+- Has to conform the same convention as Kubernetes namespaces additionally (i. e. the [DNS label names convention](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)), which means:
+    - contain only lowercase alphanumeric characters or '-'
+    - start with an alphanumeric character
+    - end with an alphanumeric character
+
+Since there will be a namespace created for each organization, prefixed with `org-`, we recommend against using that same prefix in the organization name, to avoid confusion.
+
+## Organization namespace {#namespace}
+
+For each organization there is a namespace created in the management cluster. The namespace is named after the organization CR name, prefixed with `org-`.
+
+For example, for an organization `acme`, there is the defining organization CR named `acme`. In addition, organization-operator ensures the existence of the namespace `org-acme` in the management cluster.
```

Comment:
```suggestion
For example, for an organization `acme`, there is the defining organization CR named `acme`. In addition, [organization-operator](https://github.com/giantswarm/organization-operator) ensures the existence of the namespace `org-acme` in the management cluster.
```

Reply:
It's already linked further in the top. I dont't think it's necessary to link every occurence.